lib-local/edu/harvard/huit/lts/ots/1.0.62/ots-1.0.62.pom

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Compare this pom to the 1.0.61 pom. At the minimum, this pom needs to model its dependencies.

Reply:
Sorry, I don't understand. In the past I've always updated by removing the previous OTS entry since it's no longer valid. Feel free to update as you see fit.

Reply:
Yeah, it's fine to remove the old artifact. The issue I'm pointing out is that the new pom is an auto-generated pom that does not model its dependencies.

Reply:
Yes, understood. I use the command in the _readme.txt I put into the lib-local directory to install the artifact locally. Not sure how to go about what you are suggesting so feel free to update appropriately so I can learn what would be a best practice.

Reply:
The easiest thing to do is:

```shell
mvn install:install-file -Dfile=<path-to-file> -DpomFile=<path-to-pomfile>
```

Where `<path-to-pomfile>` is the path to the actual ots pom

Reply:
If by `<path-to-pomfile>` means the file in my .m2 repo, I get an error.
`mvn install:install-file -Dfile=./lib-local/ots-1.0.62.jar -DpomFile=/Users/dneiman/.m2/repository/edu/harvard/huit/lts/ots/1.0.62/ots-1.0.62.pom
`
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-install-plugin:2.4:install-file (default-cli) on project fits-parent: Error installing artifact 'edu.harvard.huit.lts:ots:jar': Failed to install artifact edu.harvard.huit.lts:ots:pom:1.0.62: cannot install /Users/dneiman/.m2/repository/edu/harvard/huit/lts/ots/1.0.62/ots-1.0.62.pom to same path -> [Help 1]

If by `<path-to-pomfile>` means the file in `lib-local/edu/harvard/huit/lts/ots/1.0.62/ots-1.0.62.pom` then there has been no change to what I've already done.

Reply:
The former. You have to delete the artifact you already installed.

Reply:
So why can't we keep things the way they are since they work?
The same process has been used for the other JAR files in `lib-local`.

Reply:
Also include `-DlocalRepositoryPath=./lib-local`

Reply:
> So why can't we keep things the way they are since they work? The same process has been used to install the other JAR files in `lib-local`.

The old process only works so long as FITS already includes all of the dependencies that the local dependency you're trying to add needs. It is not a good way to do things.

Reply:
This succeeded:
` mvn install:install-file -Dfile=./ots-1.0.62.jar -DpomFile=/Users/dneiman/.m2/repository/edu/harvard/huit/lts/ots/1.0.62/ots-1.0.62.pom  -DlocalRepositoryPath=./lib-local`
However, there was NO change to `./lib-local/edu/harvard/huit/lts/ots/1.0.62/ots-1.0.62.pom` even when I deleted that parent directory.

Reply:
Okay, I'll take care of it this evening

Reply:
Turns out the POM in my .m2 was abbreviated but when updating from the remote LTS repo it now has all of it's dependencies which then gets updating in FITS. Will commit to this PR.

Reply:
Updates pushed. Tests via Docker all passed locally.